docs/concepts/serialization.md

```diff
@@ -704,18 +704,37 @@ print(outer_model.model_dump(serialize_as_any=False))  # (2)!
 
 ## Field inclusion and exclusion
 
-Model fields can be excluded for the serialization output in two ways:
+For serialization, field inclusion and exclusion can be configured in two ways:
 
-* using the `exclude` parameter on [the `Field()` function](fields.md).
+* at the field level, using the `exclude` and `exclude_if` parameters on [the `Field()` function](fields.md).
 * using the various serialization parameters on the [serialization methods](#serializing-data).
 
-Setting `exclude` to `True` on the [`Field()`][pydantic.fields.Field] function will unconditionally exclude it from
-the output (and this setting takes priority over the runtime parameters described below).
+### At the field level
+
+At the field level, the `exclude` and `exclude_if` parameters can be used:
+
+```python
+from pydantic import BaseModel, Field
+
+
+class Transaction(BaseModel):
+    id: int
+    private_id: int = Field(exclude=True)
+    value: int = Field(exclude_if=lambda v: v > 0)
+
+
+print(Order(id=1, private_id=2, value=0).model_dump())
+#> {'id': 1}
```

Comment:
This examples looks wrong to me. Shouldn't it

```suggestion
class Transaction(BaseModel):
    id: int
    private_id: int = Field(exclude=True)
    value: int = Field(exclude_if=lambda v: v <= 0)


print(Order(id=1, private_id=2, value=0).model_dump())
#> {'id': 1}
```

Reply:
Nice catch, updated.